requirements.txt

```
@@ -1,5 +1,6 @@
 catboost
 category-encoders
+dask
```

I'm guessing you'll need
```suggestion
dask[dataframe]
```

---

Dask core doesn't include dataframe dependecies